ipinfo/handler.py

```diff
@@ -210,6 +210,14 @@ def getBatchDetails(
             ):
                 ip_address = ip_address.exploded
 
+            if ip_address and is_bogon(ip_address):
+                details = {}
+                details["ip"] = ip_address
+                details["bogon"] = True
+                return Details(details)
```

Comment:
This is clearly broken.

Reply:
Can give me little more information about this, it will help me identify and analyze the cause.

Reply:
It is returning the details